assemblyline/odm/models/submission.py

```diff
@@ -114,46 +120,49 @@ def create_filescore_key(self, sha256, services: list = None):
         return 'v'.join([str(hashlib.md5(s.encode()).hexdigest()), str(version)])
 
 
-@odm.model(index=True, store=True, description="Submission-Relevant Times")
+@odm.model(index=True, store=True, description="""Captures important timestamps related to the submission's lifecycle, such as when it was submitted and when the analysis was completed.
+""")
 class Times(odm.Model):
-    completed = odm.Optional(odm.Date(store=False), description="Date at which the submission finished scanning")
-    submitted = odm.Date(default="NOW", description="Date at which the submission started scanning")
+    completed = odm.Optional(odm.Date(store=False), description="Timestamp recording when the analysis of the submission was completed.")
+    submitted = odm.Date(default="NOW", description="Timestamp marking when the submission was initiated and began the analysis process.")
 
 
-@odm.model(index=True, store=False, description="Submission Verdict")
+@odm.model(index=True, store=False, description="""Reflects the consensus on whether the submission is deemed malicious or non-malicious based on user input.
+""")
 class Verdict(odm.Model):
-    malicious = odm.List(odm.Keyword(), default=[], description="List of user that thinks this submission is malicious")
+    malicious = odm.List(odm.Keyword(), default=[], description="A list of usernames representing users who have judged the submission to be malicious.")
     non_malicious = odm.List(
         odm.Keyword(),
         default=[],
-        description="List of user that thinks this submission is non-malicious")
+        description="A list of usernames representing users who have judged the submission to be non-malicious.")
 
 
-@odm.model(index=True, store=True, description="Model of Submission")
+@odm.model(index=True, store=True, description="""A Submission in Assemblyline is a critical entity that encapsulates the analysis process and data for a file or collection of files submitted to determine the presence of malware. It contains a wealth of attributes such as file details, parameters for analysis, and the outcome of the scan. Familiarity with the Submission model is essential for users who need to construct precise Lucene search queries. This understanding enables them to effectively navigate and query the Assemblyline system to obtain detailed information on submissions, streamline their search efforts, and efficiently access the desired analysis results.
```

Comment:
We can remove `Familiarity with the **<OBJECT>** model is essential for users who need to construct precise Lucene search queries. This understanding enables them to effectively navigate and query the Assemblyline system to obtain detailed information on **<OBJECT>**, streamline their search efforts, and efficiently access the desired analysis results.`
It does not describe this specific object, and it true for a lot of objects, which would bloat the documentation by a lot.

Reply:
Totally agree with you, but for now, that's the best way I've found to hint the LLM to use those documentation pages when it comes to crafting Lucene queries.  But I am not happy with this and I will look for a better way.  So for now, I would leave those in and I will come back later for an update once I find a better way.